third_party/dcurl/BUILD.bzl

```diff
@@ -0,0 +1,101 @@
+licenses(["notice"])  # MIT license
+
+config_setting(
+    name = "avx",
+    values = {"define": "BUILD_AVX=1"},
+)
+
+config_setting(
+    name = "stat",
+    values = {"define": "BUILD_STAT=1"},
+)
+
+ISA = select({
+    "avx": ["-mavx -DENABLE_AVX"],
+    "//conditions:default": ["-msse2 -DENABLE_SSE"],
+})
+
+DEPS = select({
+    "avx": ["pow_avx"],
+    "//conditions:default": ["pow_sse"],
+})
+
+cc_library(
+    name = "dcurl",
+    copts = ISA + ["-O3"],
```

Comment:
I'm thinking the use of [-march=native or -mtune=native](https://wiki.gentoo.org/wiki/GCC_optimization). However, it should be forbidden with cross compilation.

Reply:
It's not ideal to put -O3 here either. It would counter bazel's [compilation mode](https://docs.bazel.build/versions/master/user-manual.html) which only supports limited options. Further configuration requires [TOOLCHAINS](https://docs.bazel.build/versions/master/toolchains.html) and CROSSTOOLS. I think this PR may not meet desired requirements we want. Maybe we can consider other options like make dcurl submodule(similar to swarm node in python) before start building tanagle-accelerator?